### ISSUE TITLE
Buffer.alloc - type string

### ISSUE THREAD

**Issue by redgumnfp** [Levminer/speakeasy#5]
Hey,

Came across this new fork, but found an issue when using it:
https://codesandbox.io/s/boring-hertz-tcdyy?file=/src/index.js

Looks like the new buffer.alloc function requires a bit of refactoring to work.

TypeError [ERR_INVALID_ARG_TYPE]: The "size" argument must be of type number. Received type string ('@q#t7,e?x@aA]iWta%5I8&iL...)

Regards,
James

**Comment by Levminer** [comment]
Hi @redgumnfp!
Thanks for reporting. Can you provide some more context? I'm not seeing your code on the link you provided.

**Comment by redgumnfp** [comment]
Oh, helps if I press save.... sorry - there on the link now.

```
const speakeasy = require("@levminer/speakeasy");

const secret = speakeasy.generateSecret().base32;

console.log(secret);

```

**Comment by Levminer** [comment]
Okay @redgumnfp, thanks for the issue, this is fixed in 1.0.7!
